backend/e4h-services/ingestion-service/app/utils/fieldplan_service_client.py

```
@@ -49,27 +50,46 @@ def create_fieldPlan_facility(self, request_info: RequestInfo, fieldPlan_id: str
-                    "tenantId": "in"
-                }
-                for facility_id in facility_ids
-            ]
+            "FieldPlanFacilities": field_plan_facilities,
         }
-        logger.trace(f"Bulk creating field plan facilities: fieldplan_id={fieldPlan_id}, count={len(facility_ids)}")
+        logger.trace(
+            f"Bulk creating field plan facilities: fieldplan_id={fieldPlan_id}, count={len(facilities)}"
+        )
         try:
             response = requests.post(url, headers=headers, json=payload)
```

Ruff (0.15.20)

[error] Probable use of `requests` call without timeout

(S113)